src/audio/Audio.js

```diff
@@ -37,6 +37,10 @@ class Audio extends Object3D {
 
 	}
 
+	clone( recursive ) {
+		return new this.constructor( this.listener.clone( recursive ) ).copy( this, recursive );
```

Comment:
The policy in the engine is to never overwrite `Object3D.clone()`. It's necessary to refactor `Audio` so an overwritten `copy()` can be implemented.

Reply:
Do you mind describing your use case for cloning an audio?

Reply:
@Mugen87 hi mugen, sorry for my holiday delay. In my own project, I was trying to encapsulate three.js with some easy-to-use scripting interfaces (for game), by the design of game logic, a `clone()` method is frequently called on all kinds of objects.
I will try to have an overwritten `copy()` with some examples later. Thanks!